litgpt/data/__init__.py

```diff
@@ -4,7 +4,7 @@
 from litgpt.data.alpaca import Alpaca
 from litgpt.data.alpaca_2k import Alpaca2k
 from litgpt.data.alpaca_gpt4 import AlpacaGPT4
-from litgpt.data.json import JSON
+from litgpt.data.json_utils import JSON
```

Comment:
I find utils not suitable here because we already have a utils.py for other purposes. How about `json_data`?

Reply:
That's a good idea. Renamed it!